rasa/core/training/story_reader/yaml_story_reader.py

```diff
@@ -161,17 +156,16 @@ def _parse_raw_user_utterance(self, step: Dict[Text, Any]) -> Optional[UserUtter
         raw_entities = step.get(KEY_STORY_ENTITIES, [])
         final_entities = YAMLStoryReader._parse_raw_entities(raw_entities)
 
-        if not user_utterance.startswith(INTENT_MESSAGE_PREFIX):
+        if user_utterance.startswith(INTENT_MESSAGE_PREFIX):
             common_utils.raise_warning(
                 f"Issue found in '{self.source_name}': \n"
-                f"User intent '{user_utterance}' should start with "
-                f"'{INTENT_MESSAGE_PREFIX}'. "
-                f"This story step will be skipped.",
+                f"User intent '{user_utterance}' starts with "
+                f"'{INTENT_MESSAGE_PREFIX}'. This is not required.",
```

Comment:
should we phrase that differently?

Reply:
The phrasing is fine. I'm just thinking what if users will accidentally confuse `intent` key with a `user` one?
Should we check if that's a single word after stripping the spaces?